README.md

```diff
@@ -335,6 +335,14 @@ Each command can be executed with `invoke <command>`. All commands support the a
   unittest         Run Django unit tests for the plugin.
 ```
 
+`unittest` uses by default the Postgres DB backend, if you want to test Mysql you must use Invoke environment variables:
```

Comment:
This isn't specific to `invoke unittest` - all of the development environment will use postgres by default. As such I think it'd be better to move this addition up to the parent `CLI Helper Commands` section.